larq_compute_engine/mlir/tf_tfl_passes.cc

```diff
@@ -131,6 +127,14 @@ void AddTFToLCETFLConversionPasses(
   // constant ops.
   pass_manager->addPass(mlir::tf_saved_model::CreateFreezeGlobalTensorsPass());
 
+  // Reduce operands of TFL::While without changing the outcome.
```

Comment:
I tried to synchronize this file here as much as possible with the latest changes. That means some new passes have been added. I did not check what they do and therefore are not sure that we need/want them in. This should perhaps be checked?

Reply:
I double checked and could only find a minor difference. Fixed in 114a945b7670d0ffe96678f81d4b72bfbb4a0e09